cmd/respawn/respawn.go

```diff
@@ -69,6 +69,12 @@ func run(c *cli.Context) error {
 		panic(err)
 	}
 
+	// Now that we're booted, stop writing debug messages to the console
```

Comment:
Can this go in `console_init` instead? I would think to look there for something like this.